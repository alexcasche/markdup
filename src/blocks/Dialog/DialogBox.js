import styled from 'styled-components'
import { tint } from 'polished'

const DialogBox = styled.div`
  position: relative;
  left: 50%;
  transform: translateX(-50%);
  margin-top: 60px;
  padding: 30px;
  width: calc(100% - 40px);
  max-width: 600px;
  text-align: right;
  background-color: ${props => tint(0.85, props.theme.colors.black)};
  box-shadow: 0 2px 10px 0 ${props => props.theme.colors.black};
  border-radius: 5px;
`

export default DialogBox
