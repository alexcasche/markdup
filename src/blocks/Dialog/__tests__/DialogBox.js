import DialogBox from '../DialogBox'

describe('DialogBox Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogBox />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
