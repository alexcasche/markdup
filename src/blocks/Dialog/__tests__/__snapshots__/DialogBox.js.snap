// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogBox Element renders correctly 1`] = `
.c0 {
  position: relative;
  left: 50%;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
  margin-top: 60px;
  padding: 30px;
  width: calc(100% - 40px);
  max-width: 600px;
  text-align: right;
  background-color: #3e4042;
  box-shadow: 0 2px 10px 0 #1D1F21;
  border-radius: 5px;
}

<div
  className="c0"
/>
`;
